docs/getting-started/cli-oss.md

```diff
@@ -0,0 +1,96 @@
+---
+slug: cli
+title: CLI-only workflow (OSS)
+hide_title: true
+description: Learn how to set up Semgrep, scan your first project for security issues, and view your findings in the CLI.
+tags:
+  - quickstart
+  - cli
+---
+
+import MoreHelp from "/src/components/MoreHelp";
+import Install from "/src/components/procedure/_install-cli.mdx";
+import Login from "/src/components/procedure/_login-activate.mdx";
+
+# CLI-only workflow with Semgrep's OSS Engine
+
+Learn how to set up Semgrep, scan your first project for security issues, and view your findings in the CLI.
+
+## Prerequisites
+
+See [Prerequisites](/prerequisites/) to ensure that your machine meets Semgrep's requirements.
+
+## Set up Semgrep
+
+<Install />
+
+## Scan your project
+
+# Semgrep OSS
+
+Navigate to the root of your repository, and run your first scan. The specific command you use depends on how you want to view the results.
+
+To view the results in the CLI:
+
+```console
+semgrep scan
+```
+
+To export the results to a SARIF file:
+
+```console
+semgrep scan --sarif --output=semgrep.sarif
+```
+
+To export the results to a JSON file:
+
+```console
+semgrep scan --json --output=semgrep.json
+```
+
+### Scan your project with a specific ruleset
+
+You can scan your project using `--config auto` to automatically obtain rules tailored to your project:
```

Comment:
```suggestion
You can scan your project using `--config auto` to run Semgrep with rules that apply to your to your codebase:
```
Let's stick to the ff definitions:

codebase - more general term ("broader term"); anything scanned by semgrep; does not need to be a git repo 
repository - same definition as ... repository (git is implied)
project - any repos onboarded to Semgrep Cloud Platform, as that is the term used in the UI.